.github/test_dependencies.yaml

```diff
@@ -1,26 +1,40 @@
 e2e:
   kind:
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.28.0'
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.27.3'
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.26.6'
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.25.11'
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.24.15'
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.23.17'
   gke:
     - 'v1.26.5'
+
+  # For Istio, we define combinations of Kind and Istio versions that will be
+  # used directly in the test matrix `include` section.
   istio:
-    # For Istio, we define combinations of Kind and Istio versions that will be
-    # used directly in the test matrix `include` section.
     - kind: 'v1.28.0'
-      istio: 'v1.19'
+      # renovate: datasource=docker depName=istio/istioctl versioning=docker
+      istio: 'v1.19.0'
     - kind: 'v1.27.3'
-      istio: 'v1.18'
+      # renovate: datasource=docker depName=istio/istioctl versioning=docker
+      istio: 'v1.18.0'
     - kind: 'v1.26.6'
-      istio: 'v1.17'
+      # renovate: datasource=docker depName=istio/istioctl versioning=docker
+      istio: 'v1.17.0'
     - kind: 'v1.25.11'
-      istio: 'v1.16'
+      # renovate: datasource=docker depName=istio/istioctl versioning=docker
+      istio: 'v1.16.0'
 
 integration:
+  # renovate: datasource=docker depName=kindest/node versioning=docker
   kind: 'v1.28.0'
-  kong-oss: '3.4'
-  kong-ee: '3.4'
+  # renovate: datasource=docker depName=kong versioning=docker
+  kong-oss: '3.4.0'
```

Comment:
Why `3.4.0` here? `kong` already has `3.4.1` tag. (and 3.4.1 is required for KIC 3.0)

Reply:
I believe this will be done by renovate. PTAL at the PR description

<img width="525" alt="image" src="https://github.com/Kong/kubernetes-ingress-controller/assets/739996/3c4328b2-a14f-4ac1-9409-b72a4a78e710">